h/groups/__init__.py

```diff
@@ -1,5 +1,9 @@
 # -*- coding: utf-8 -*-
 
+from h.groups.auth import annotation_group_write_permitted
+
 
 def includeme(config):
+    config.set_memex_group_write_permitted(annotation_group_write_permitted)
```

Comment:
What is the best way to test this? Not having this call tested makes me a bit nervous since the default permission then is permissive.